test/core/miner/sealing_test.cpp

```diff
@@ -426,6 +426,7 @@ namespace fc::mining {
   TEST_F(SealingTest, MarkForUpgradeSeveralPieces) {
     UnpaddedPieceSize piece_size(127);
     PieceData piece("/dev/random");
+    PieceData piece2("/dev/random");
```

Comment:
Why do you need `piece2` with the same value with `piece`?
And I don't see anymore where `piece` is used.

Reply:
Because PieceData is move-only and this test need two pieces.

Reply:
But piece is not used

Reply:
Line 473